### PR TITLE
404 fix

### PR DESCRIPTION
**Pull request by VanhDeepTryy**
Just for fix 404 errors.

-[@NDKDD](https://discord.com/users/712638495310479442)-

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,17 +5,17 @@
 - [**Logo**](https://github.com/discordexperimenthub/deh-assets/blob/main/svg/logo.svg) by [@IllagerCaptain](https://github.com/IllagerCaptain)
 - [**Primary Banner**](https://github.com/discordexperimenthub/deh-assets/blob/main/svg/banner_primary.svg) by [@Pukimaa](https://github.com/Pukimaa)
 - [**Secondary Banner**](https://github.com/discordexperimenthub/deh-assets/blob/main/svg/banner_secondary.svg) by [@Lordcobcob](https://github.com/Lordcobcob)
-- [**Role Icons**](https://github.com/discordexperimenthub/deh-assets/tree/main/svg/Role%2520Icons) by [@Pukimaa](https://github.com/Pukimaa)
+- [**Role Icons**](https://github.com/discordexperimenthub/deh-assets/tree/main/svg/Role%20Icons) by [@Pukimaa](https://github.com/Pukimaa)
```
Comment (Tolga1452) [inline README.md:8]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -5,17 +5,17 @@
...
 - [**Stickers**](https://github.com/discordexperimenthub/deh-assets/tree/main/svg/Stickers) by [@Pukimaa](https://github.com/Pukimaa)
 
-### [Test Server Icons](https://github.com/discordexperimenthub/deh-assets/tree/main/svg/Role%2520Icons)
+### [Test Server Icons](https://github.com/discordexperimenthub/deh-assets/tree/main/svg/Role%20Icons)
```
Comment (Tolga1452) [inline README.md:11]:
```suggestion
### [Role Icons](https://github.com/discordexperimenthub/deh-assets/tree/main/svg/Role%20Icons)
```

**Review by Tolga1452 — CHANGES_REQUESTED** [review]
Thanks! But looks like i missed some more parts, can you please fix them too?

File: README.md (hunk 2 above)
Comment (VanhDeepTryy) [reply README.md:10]: Done. 
-[@NDKDD](https://discord.com/users/712638495310479442)-

**Review by Tolga1452 — APPROVED** [review]

**Pull request merged**